src/Meilisearch/Builder/FilterBuilderInterface.php

```diff
@@ -6,5 +6,7 @@
 
 interface FilterBuilderInterface
 {
-    public function build(array $parameters): array;
+    public function build(array $facets): string|array;
```

Comment:
We could potentially unify the return type, to always return array (no matter if with one or multiple elements)